frappe/utils/install.py

```diff
@@ -5,11 +5,11 @@
 from frappe.utils.password import update_password
 
 def before_install():
+	frappe.reload_doc("core", "doctype", "doctype_state")
```

Comment:
note: this ordering is required for Postgres because reloading other docs causes querying doc_state which doesn't exist yet.